src/LottoManager.js

```diff
@@ -0,0 +1,32 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import Lotto from './Lotto.js';
+import { OUTPUT_MESSAGE } from './lib/constant.js';
+
+class LottoManager {
+  #lottos = [];
+
+  constructor(buyPrice) {
+    this.lottoCounts = buyPrice / 1000;
+    this.genenrateLotto(this.lottoCounts);
```

Comment:
`lottoCounts`는 `this.generateLotto()`의 파라미터로만 쓰이는 것 같은데, `const`로 변수선언을 하신게 아니라 프로퍼티로 담으신 이유가 있으신가요?
```suggestion
    const lottoCounts = buyPrice / 1000;
    this.genenrateLotto(lottoCounts);
```

Reply:
`LottoManager` 객체가 가지고 있어야하는 속성이라고 생각했는데, 지예님 말씀을 듣고보니 다른 메서드나 외부에서 접근할 일도 없고 `generateLotto`에 전달되기 위한 임시값이기에 `const`로 선언하는게 더 적절했을 것 같네요
이번에 객체를 활용하려고 의식하다보니 전반적으로 모두 프로퍼티로 작성던 것 같습니다😅